shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/test/java/org/apache/shardingsphere/sql/parser/sql/dialect/handler/ddl/DropTableStatementHandlerTest.java

```diff
@@ -30,42 +31,52 @@
 public final class DropTableStatementHandlerTest {
     
     @Test
-    public void assertContainsIfExistsForMySQL() {
+    public void assertIfExistsForMySQL() {
         assertTrue(DropTableStatementHandler.ifExists(new MySQLDropTableStatement(true)));
+        assertFalse(DropTableStatementHandler.ifExists(new MySQLDropTableStatement(false)));
     }
     
     @Test
-    public void assertContainsIfExistsForPostgreSQL() {
+    public void assertIfExistsForPostgreSQL() {
         assertTrue(DropTableStatementHandler.ifExists(new PostgreSQLDropTableStatement(true, false)));
+        assertFalse(DropTableStatementHandler.ifExists(new PostgreSQLDropTableStatement(false, false)));
     }
     
     @Test
-    public void assertContainsIfExistsForSQLServer() {
+    public void assertIfExistsForSQLServer() {
         assertTrue(DropTableStatementHandler.ifExists(new SQLServerDropTableStatement(true)));
+        assertFalse(DropTableStatementHandler.ifExists(new SQLServerDropTableStatement(false)));
     }
-    
```

Comment:
Please execute `mvn spotless:apply` to format code.

Reply:
Thanks, fixed it here - https://github.com/apache/shardingsphere/pull/20801/commits/614a7f6e17f68474c718cdc54b93ac8ef733abf9.
From now onwards, I shall run this before submitting a PR.